lib/tlog/es_json_reader.c

```diff
@@ -48,6 +48,7 @@ struct tlog_es_json_reader {
     size_t                      idx;        /**< Index of the message to be
                                                  read next */
     size_t                      last_id;    /**< Last received message ID */
+    bool                        verbose;    /**< Enable HTTP debug */
```

Comment:
We don't seem to need this beyond initialization, so we don't need to store it. Please remove.